.github/workflows/pre-commit-check.yml

```diff
@@ -13,4 +13,5 @@ jobs:
           activate-environment: pre-commit-env
           environment-file: pre-commit-environment.yml
           auto-activate-base: false
-    - uses: pre-commit/action@v3.0.0
+
+  #  - uses: pre-commit/action@v3.0.0
```

Comment:
I have the impression I recall the pre-commit test was not successful with this line included. However, I now reverted it to the master version and will wait for the new test result.

Reply:
it will fail due to formatting issues, but we can fix those at the end before the merge. You could merge `master` onto this branch then, which will make that easier. Info is on https://github.com/UCL/STIR/blob/master/documentation/devel/git-hooks.md. summary
1. install `pre-commit`
2. run `pre-commit run --all`
3. check and commit and push